experiments/convert_traj.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Convert trajectories from `imitation` format to openai/baselines GAIL format.
+"""
+
+from typing import List
+
+import numpy as np
+
+from imitation.util import rollout as ro_util
+
+
+def convert_trajs_to_sb(trajs: List[ro_util.Trajectory]) -> dict:
+  """Converts Trajectories into the dict format used by Stable Baselines GAIL.
+  """
+  trans = ro_util.flatten_trajectories(trajs)
+  return dict(
+    acs=trans.acts,
+    rews=trans.rews,
+    obs=trans.obs,
+    ep_rets=np.array([np.sum(t.rews) for t in trajs]),
+  )
+
+
+if __name__ == "__main__":
+    import argparse
+    import os
+    from pathlib import Path
+    import pickle
+    parser = argparse.ArgumentParser()
```

Comment:
Put the body in a separate function, e.g.`main()`. Right now all these variables are module-level variables. https://google.github.io/styleguide/pyguide.html#317-main